db/compaction.cc

```diff
@@ -461,4 +461,17 @@ bool Compaction::ShouldFormSubcompactions() const {
   }
 }
 
+uint64_t Compaction::MaxInputFileCreationTime() const {
+  uint64_t max_creation_time = 0;
+  if (cfd_->ioptions()->compaction_style == kCompactionStyleFIFO) {
```

Comment:
we put it there for all flushes, so maybe do it for all compactions, too? We supposedly support migrating between compaction styles so it may be useful.